internal/serve/httphandler/disbursement_handler_test.go

```diff
@@ -97,7 +97,8 @@ func Test_DisbursementHandler_PostDisbursement(t *testing.T) {
 		{
 		  "wallet_id": "aab4a4a9-2493-4f37-9741-01d5bd31d68b",
 		  "asset_id": "61dbfa89-943a-413c-b862-a2177384d321",
-		  "country_code": "UKR"
+		  "country_code": "UKR",
+		  "verification_type": "date_of_birth"
```

Comment:
shouldn't this be `verification_field` ?

Reply:
fixed this!